mockcpu_test.go

```diff
@@ -103,6 +103,10 @@ func mockCPU(def []byte) func() {
 				return 0, 0, 0, 0
 			}
 		}
+		if op == 0x1 {
```

Comment:
This shouldn't really be needed. If it is, I suspect that there may be some compatibility issues.